src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public ColorOfBall getRandomColor() {
+        int index = new Random().nextInt(ColorOfBall.values().length);
+        return ColorOfBall.values()[index];
```

Comment:
Calling `ColorOfBall.values()` inside a loop or a method that is called multiple times can be inefficient, as it creates a new array each time. Consider storing the result of `ColorOfBall.values()` in a constant field.